src/DotVVM.Core/DotVVM.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <VersionPrefix>4.0.0-preview01-116912</VersionPrefix>
-    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
```

Comment:
net472 should be compatible with netstandard2.0, right? So could we get rid of explicit target for net472?

Reply:
I tried to remove net472 but unfortunately, there is some problem with the build. I'm not completely sure whether it is possible to simply remove the net4xx completely.

Reply:
Ok, I have fixed the issue with the build but still, there is a problem with loading/referencing assemblies. It seems like we have to leave there the net472 and most probably we have to discourage people from using [net461-net471].

Reply:
The problem is still that you'll be able to install dotvvm on net461 since it support netstandard2.0...